test/xpu/fin_grain/test_ops_xpu.py

```diff
@@ -133,6 +133,8 @@
     "max_pool2d_with_indices_backward",
     "nn.functional.embedding",
     "nn.functional.unfold",
+    "_upsample_nearest_exact1d",
```

Comment:
It should not be the name of op in OPDB. Please check https://github.com/pytorch/pytorch/blob/9a8ab778d34bd24c5caceb340837483decc4c311/torch/testing/_internal/common_methods_invocations.py#L11588 for the name you need.